.github/workflows/integration-tests.yml

```diff
@@ -16,12 +16,13 @@ on:
 env:
   GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-# As of 16 June 2023, ubuntu-latest, windows-latest and macos-latest come
-# with Stack 2.11.1 and GHC 9.6.2. windows-latest comes with NSIS 3.08, for
-# which the default value of the 'Unicode' installer attribute is 'true'.
-# However, that is not the 'large strings' build of NSIS and creates installers
-# that corrupt the PATH environment variable if the default string length of
-# 1024 characters is exceeded.
+# As of 3 July 2023, ubuntu-latest, windows-latest come with Stack 2.11.1 and
+# GHC 9.6.2. macos-13 comes with Stack 2.9.3 and GHC 9.6.2 but does not come
+# pkg-config. windows-latest comes with NSIS 3.08, for which the default value
```

Comment:
Noticed a typo: "does not come __with__ pkg-config"

Reply:
Thanks! The context of this PR (in #6182) is that @essandess can't build Stack 2.11.1 on macOS/x86_64 in a MacPorts environment. It builds fine on GitHub actions 'plain vanilla' macOS environment. This was to rule out (using CI) that it was somehow a 'macOS 12 v macOS 13' thing. I'm not planning to actually merge the PR.